services/ui-src/src/components/pages/Export/McparPdfExport.tsx

```diff
@@ -0,0 +1,126 @@
+import { Box, Heading } from "@chakra-ui/react";
+import { StickyBanner, ReportContext, Table, FieldsSection } from "components";
+
+import { useContext } from "react";
+import { convertDateUtcToEt } from "utils";
+
+import addEditProgram from "../../../forms/addEditProgram/addEditProgram.json";
+
+export const McparPdfExport = () => {
```

Comment:
Suggestion: Change name to `ExportedReportPage` or something like that since we'll be using this for other reports, not just MCPAR.